its/ruling/src/test/resources/expected/python-S6911.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Are these TPs in your opinion?
Acceptable FPs? Non acceptable FPs?

If the issues are not TPs I'd expect a comment mentioning that you've reviewed them and that you think they're acceptable. If they're not, we should either reconsider the rule or at least its implementation.

I'm saying this because I have the impression these are FPs since these all are Tensorflow / Keras objects that are involved. Maybe the issue is still warranted as using parameters would be cleaner, though. But since the rule is about reliability of the execution, it's still not as bad as the rule suggests.